lib/platform/common.ts

```diff
@@ -3,6 +3,11 @@ import Git from 'simple-git/promise';
 import { RenovateConfig } from '../config/common';
 import { BranchStatus } from '../types';
 
+export interface Comment {
```

Comment:
We need to move the `Comment` interface here, because we now export the `getComments` method on the `Platform` interface: https://github.com/renovatebot/renovate/pull/6002/files#diff-660268298abca8fe83745a74ee3a6802R199